libs/cart/src/models/cart-item.ts

```diff
@@ -6,6 +6,7 @@ import { DaffCartItemInputType } from './cart-item-input';
 
 export interface DaffCartItem {
 	item_id: number | string;
+	state: DaffCartItemStateEnum;
```

Comment:
Can you document this field and explain that its just for decaying (client-side) states?

Reply:
added